okapi-core/src/main/java/org/folio/okapi/util/DepResolution.java

```diff
@@ -128,63 +128,59 @@ public static void getLatestProducts(int limit, List<ModuleDescriptor> mdl) {
 
   /**
    * Check if md's required and optional interfaces are provided by set of modules.
```

Comment:
This implementation doesn't check optional interfaces.

Reply:
Correct. Fixed now!